eval-bot/src/main/resources/sbot-defaults.cfg

```diff
@@ -1 +1,5 @@
-entityTimeout  = 30 seconds
+
+requiredMessagePrefix = ""
+
+http.entityTimeout    = 30 seconds
```

Comment:
I think we should follow the typesafe config conventions here. `reference.conf` for libs with reasonable defaults that can easily be overriden and `application.conf` for the app config. https://github.com/typesafehub/config#standard-behavior